docs-website/topics/serialization.md

```diff
@@ -0,0 +1,72 @@
+[//]: # (title: Serialization)
```

Comment:
`docs-website/images/serialization.svg` is no longer needed, so you can put it aside